main.go

```diff
@@ -657,6 +657,14 @@ func (o *Options) runServerMode(pinnedDateTime *time.Time, gormLogLevel gormlogg
 		return err
 	}
 
+	gcsClient, err := gcs.NewGCSClient(context.TODO(),
+		o.GoogleServiceAccountCredentialFile,
+		o.GoogleOAuthClientCredentialFile,
+	)
+	if err != nil {
+		log.WithError(err).Warn("unable to create GCS client, some APIs may not work")
```

Comment:
If bigquery support will be optional going forward we should maybe use the `capabilities` feature in pkg/sippyserver/server.go, but I expect it won't be optional much longer